core/primitives/piece/piece_data.hpp

```diff
@@ -28,8 +29,11 @@ namespace fc::primitives::piece {
 
     bool isOpened() const;
 
+    bool IsNullData() const;
```

Comment:
Name functions with small letter, like the all other functions in the class.
```suggestion
    bool isNullData() const;
```

Reply:
Do we have any agreement of that?
Because I thought we should name methods of classes starting with capital letter. And use small letter for functions outside any class or struct.

Reply:
Yes, we have some agreements. Class names should start with capital Letter and camel case used. Functions names: camelcase from small letter. class members and variables snake case starting from small letter.

Reply:
Fixed